alert.css

```diff
@@ -0,0 +1,65 @@
+.alert {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.alert p {
+  margin-left: 0.7rem;
+}
+
+.success-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--alertgreen);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
+}
+.warning-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--alertyellow);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
+}
+.error-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--flatred);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
+}
+.feature-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--flatblue);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
+}
```

Comment:
Extract the common styles into another class, it will reduce redundancy. Here it's repeated in all the styles. You can extract the common ones- font size,width,display,gap, flex wrap, text align

Reply:
Done